src/modules/45-dashboards/components/DashboardCard/__tests__/DashboardCard.test.tsx

```diff
@@ -83,6 +83,7 @@ describe('DashboardCard', () => {
     jest
       .spyOn(customDashboardServices, 'useSearchFolders')
       .mockImplementation(() => ({ data: mockGetFolderResponse } as any))
+    jest.spyOn(customDashboardServices, 'useGetModelTags').mockImplementation(() => ({ data: { resource: [] } } as any))
```

Comment:
Nothing wrong here, but you can simplify a wee bit with `mockReturnValue`
```suggestion
    jest.spyOn(customDashboardServices, 'useGetModelTags').mockReturnValue({ data: { resource: [] } } as any)
```